internal/v1/api.yaml

```diff
@@ -433,6 +436,8 @@ components:
         request: {}
         created_at:
           type: string
+        image_name:
```

Comment:
By setting the image_name as an optional return value the type becomes `*string` This seems to cause issues returning the string. @Gundersanne do you have any thoughts about what I'm doing wrong here?

Reply:
You have to return `*string` in this case. I would help more but I need to see the error and the line of code that's causing it.

Reply:
I guess my confusion about returning *string is that wouldn't that return a pointer to the user?